src/main/java/org/commcare/formplayer/services/FormplayerLockRegistry.java

```diff
@@ -88,15 +88,15 @@ public FormplayerReentrantLock obtain(Object lockKey) {
 
     private void evict(FormplayerReentrantLock lock, Object lockKey) {
         Thread ownerThread = lock.getOwner();
-        log.error(String.format("Thread %s owns expired lock with lock key %s.", ownerThread, lockKey));
+        log.warn(String.format("Thread %s owns expired lock with lock key %s.", ownerThread, lockKey));
         ownerThread.interrupt();
         try {
             ownerThread.join(5000);
         } catch (InterruptedException e) {
             throw new InterruptedRuntimeException(e);
         }
         if (ownerThread.isAlive()) {
-            log.error(String.format(
+            log.warn(String.format(
```

Comment:
since the exception is captured below I've changed this to a 'warn' which will still output in the console but won't results in a Sentry event on it's own